core/src/main/java/com/github/jsonldjava/core/Context.java

```diff
@@ -1146,57 +1144,4 @@ else if (this.get(JsonLdConsts.LANGUAGE) != null) {
         }
         return rval;
     }
-
-    public Map<String, Object> serialize() {
```

Comment:
I see that you don't need this method anymore and find your solution good. But removing this method may be an API break, because it's public in a public class, so maybe somebody makes use of it in her code. It may be better to mark this method as deprecated and remove it in al later release. What do you think @ansell ?